packages/amplify-provider-awscloudformation/src/iterative-deployment/deployment-manager.ts

```diff
@@ -175,6 +204,8 @@ export class DeploymentManager {
               return resolve();
             case 'rolledBack':
             case 'failed':
+              this.printer.stopBars();
+              promptsPrinter.info(`Rolled back (${maxDeployed - state.context.currentIndex} of ${maxDeployed})`);
```

Comment:
probably for future. Can we replace this massive `switch-case` with event handlers ?

Reply:
I want to keep this PR scoped down to changing the terminal output.

Reply:
@jhockett yup. How do we keep track of ideas like this?